watch_faces/BinarySystem/resources/settings/properties.xml

```diff
@@ -3,11 +3,20 @@
     <property id="appVersion" type="string">2.1</property>   
     
     <property id="LayoutType" type="number">1</property>
-    <property id="ShowSeconds" type="boolean">true</property>
-    <property id="BinaryRadius" type="number">8</property>
+    
+    <property id="showBatteryPercentage" type="boolean">true</property>
+    <property id="showBatteryBar" type="boolean">true</property>
     <property id="RemainingBatteryEstimate" type="boolean">false</property>
     
-    <property id="BackgroundColor" type="number">0x000000</property>
+    <property id="showTime" type="boolean">true</property>
```

Comment:
showTime and showDate are incompatible with geekmode and non-geekmode. 
Furthermore the implementation is not in place and not used in the Layouts. I would remove the options until the implementation is less conflicting.